src/components/ItaliaTheme/View/Commons/SideMenu.jsx

```diff
@@ -127,92 +127,80 @@ const SideMenu = ({ data, content_uid }) => {
   return headers?.length > 0 ? (
     <div className="sticky-wrapper navbar-wrapper page-side-menu">
       <nav className="navbar it-navscroll-wrapper navbar-expand-lg">
-        <button
-          className={
-            isNavOpen
-              ? 'custom-navbar-toggler focus--mouse'
-              : 'custom-navbar-toggler'
-          }
-          type="button"
-          aria-controls="navbarNavB"
-          aria-expanded={isNavOpen ? 'true' : 'false'}
-          aria-label="Toggle navigation"
-          data-target="#navbarNavB"
-          onClick={() => {
-            setIsNavOpen(!isNavOpen);
-          }}
-        >
-          <span className="it-list"></span>
-          {intl.formatMessage(messages.index)}
-        </button>
-
-        <div
-          className={
-            isNavOpen ? 'navbar-collapsable expanded' : 'navbar-collapsable'
-          }
-          id="navbarNavB"
-          style={isNavOpen ? { display: 'block' } : { display: 'none' }}
-        >
-          <div
-            className="overlay"
-            style={isNavOpen ? { display: 'block' } : { display: 'none' }}
-          ></div>
-          <div className="close-div visually-hidden">
-            <button className="btn close-menu" type="button">
-              <span className="it-close"></span>
-              {intl.formatMessage(messages.close)}
-            </button>
-          </div>
-          <a
-            className="it-back-button"
-            href="#"
-            style={isNavOpen ? { display: 'block' } : { display: 'none' }}
-            onClick={(e) => {
-              e.preventDefault();
-              setIsNavOpen(!isNavOpen);
-            }}
-          >
-            <Icon
-              className="align-top"
-              color="primary"
-              icon="it-chevron-left"
-              style={{ ariaHidden: true }}
-              size="sm"
-            />
-            <span>{intl.formatMessage(messages.back)}</span>
-          </a>
-          <div className="menu-wrapper">
-            <div className="link-list-wrapper menu-link-list">
-              <h3>{intl.formatMessage(messages.index)}</h3>
-              <div className="mb-3">
-                <Progress
-                  value={
-                    100 *
-                    (window.scrollY /
-                      (document.documentElement.scrollHeight -
-                        document.documentElement.clientHeight))
+        <div className="menu-wrapper">
+          <div className="link-list-wrapper menu-link-list">
+            <div className="accordion">
```

Comment:
In design-react-kit c'è un componente Accordion che dovrebbe già gestire un po' di logica e stili e semplificare un po' il nostro codice. Non lo trovi sulla documentazione ufficiale perché fa parte della versione nuova che stanno sviluppando per il PNRR, ma @SaraBianchi ci ha già litigato. Sara ti taggo per capire se alla fine ha funzionato o ti ha dato problemi.